lib/rules/no-sparse-arrays.js

```diff
@@ -10,6 +10,8 @@
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
Not sure I understand why this is a probable bug. (Should this be "suggestion"?)